rca/scholarships/models.py

```diff
@@ -144,6 +147,14 @@ class ScholarshipsListingPage(ContactFieldsMixin, BasePage):
         ]
     )
 
+    @property
+    def show_interest_bar(self):
+        return True
+
+    @property
+    def show_interest_link(self):
+        return True
+
```

Comment:
**praise**

:+1: I take it this is in prep for an expected setting down the line